hooks/pre-commit

```diff
@@ -12,6 +12,26 @@
 #
 # This script does not handle file names that contain spaces.
 
+govet=$(go vet ./lib)
```

Comment:
Since this is a `pre-commit` hook, it may be advisable to check our staged files instead of a specific directory. One could change `server.go` (which is not in `./lib`) to fail `go vet`.